tests/integrationTests/D3DTestCase/batchFile.dat

```diff
@@ -0,0 +1,74 @@
+
+#HEAT batchFile
+#For use when running HEAT in terminal / batch mode.  Each line is a new entry.
+#
+# The fist line of every batchFile should be (uncommented):
+# MachFlag, Tag, GEQDSK, CAD, PFC, Input, Output
+#
+#===Column variables are defined as follows
```

Comment:
@wingena this comment reflects the old (v3.0) HEAT batchFile.  Copy the new comment from the NSTXU test case [here](https://github.com/plasmapotential/HEAT/blob/f6b7d74b10e56f0bb458f2bc4cec990e80a039df/tests/integrationTests/nstxuTestCase/batchFile.dat).

More specifically, the shot and timestep columns are missing.

Reply:
Thanks, I updated the comments in batch file